src/components/v1/shared/FundingCycle/FundingCycleDetails.tsx

```diff
@@ -156,13 +156,23 @@ export default function FundingCycleDetails({
 
         {fundingCycle.duration.gt(0) && (
           <Descriptions.Item label={<Trans>Start</Trans>}>
-            {formattedStartTime}
+            <Tooltip title={formatDateToUTC(fundingCycle.start.mul(1000))}>
```

Comment:
**suggestion (non-blocking)**: maybe we can create a `secondsToMilliseconds` utility for all these in a follow-up.